### ISSUE TITLE
Any reason to use a match if the last arm uses a function call with `n` anyways?

### ISSUE THREAD

**Issue by sbatial** [eza-community/eza#446]
I feel like the title explains my confusion.
If any further explanation is needed I'm happy to help.

https://github.com/eza-community/eza/blob/d08262670355d9bdf443cae49c797c54dc71c550/src/output/file_name.rs#L185-L189

As I'm [looking at the function used here](https://github.com/eza-community/eza/blob/d08262670355d9bdf443cae49c797c54dc71c550/src/output/file_name.rs#L480-L482): I'm all for code re-use and seperation of concerns and all that but... Does it make sense to have that as a distinct function in the first place?
```rust
fn spaces(width: u32) -> String {
    (0..width).map(|_| ' ').collect()
}
```
I could maybe understand a closure inside of the `paint` function to have an 'named action' (a function _spaces_ taking a number as an argument is basically documentation imo, which is the only reason I see for having this function)

---

Not a big deal really... I just stumbled across it when looking into #396 and was a bit confused.

**Comment by hehelego** [comment]
I also think this `match` is redundant. It probably won't bring any performance boost...  
That `spaces` function is probably also redundant.

**Comment by MartinFillon** [comment]
Well according to `git blame` this is some old code, so if the behaviour stays the same and everything this is clearly considerable I think. Many things in eza are not optimized due to old code owner not having one of the best way to write rust. (for exemple the arg parsing)

**Comment by sbatial** [comment]
Closed by #465
